docs/userGuide/syntax/lists.md

```diff
@@ -109,3 +109,102 @@ first number
 - [x] Item 4
 - ( ) Item 5
 </div>
+
+****Customizing the list appearance:****
+
+**To customize list icons, add the configuration `{icon="icon-name"}` either after a specific list item or at the end of the entire list.**
+
+<include src="codeAndOutput.md" boilerplate >
+<variable name="highlightStyle">markdown</variable>
+<variable name="code">
+* Item 1 {icon="glyphicon-education"}
+* Item 2
+  * Item 2.1 {icon="fas-file-code"}
+  * Item 2.2
+* Item 3 {icon="fas-code-branch"}
+  * Item 3.1 
+* Item 4 {icon="octicon-git-pull-request"}
+  * Item 4.1 {icon="mif-perm-media"}
+* Item 5 {icon="glyphicon-education"}
+  * Item 5.1 {icon="notebook_with_decorative_cover"}
+  </variable>
+</include>
+<box type="definition" seamless>
+
+1. You can use any of the icons supported by MarkBind (e.g., Font Awesome, Octicons, Glyphicons, Images, etc.) as the icon of a list item.
+1. When using emojis, omit the `::` around the icon name (e.g., use `{icon="emoji"}` instead of `{icon=":emoji:"}`).
+1. If an item has a specified icon, that icon will be used for it and for subsequent items at that level. 
+1. If any item at a level is customized, the first item at that level must also be customized. If not, the list will be invalidated.
+</box>
```

Comment:
```suggestion
1. You can use any of the [icons (e.g., Font Awesome, Octicons, Glyphicons, etc.)](#icons) or [emojis](#emoji) supported by MarkBind as the icon of a list item, but omit the `:` at the start and at the end .e.g, `fas-home` instead of `:fas-home:`.
1. If an item has a specified icon, that icon will be used for it and for subsequent items at that level e.g., `Item 2` in the example list above.
1. If any item at a level is customized, the first item at that level must also be customized. If not, the list will be considered invalid.
```

1. These are not definitions. So, `info` is a better match. 
2. Furthermore, best not to put inside a box at all so that we can use boxes for 'special occasions' only. Also, to be consistent with the rest of the UG.
3. Images are not an 'icon supported by MarkBind'